report/verification/mes/tmap_1c.md

```diff
@@ -0,0 +1,128 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# TMAP7 V&V Val-1c
+
+```{tags} 1D, MES, transient
+```
+
+This verification case {cite}`ambrosek_verification_2008` from TMAP7's V&V document consists of a semi-infinite slab with no traps under a constant concentration $C_0$ on the first $10 m$ of the slab.
+
++++
+
+## FESTIM Code
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+import festim as F
+import numpy as np
+import sympy as sp
+from matplotlib import pyplot as plt
+
+preloaded_length = 10 # m
+C_0 = 1 # atom m^-3
+D = 1 # 1 m^2 s^-1
+exact_solution = C_0/2 * (
+    2*sp.erf(F.x / sp.sqrt(4*D*F.t)) 
+    - sp.erf((F.x - preloaded_length) / sp.sqrt(4*D*F.t))
+    - sp.erf((F.x + preloaded_length) / sp.sqrt(4*D*F.t))
+)
+
+model = F.Simulation()
+
+### Mesh Settings ###
+vertices = np.concatenate([
+    np.linspace(0, 10, 200),
+    np.linspace(10, 100, 300),
+])
+
+model.mesh = F.MeshFromVertices(vertices)
+
+model.boundary_conditions = [
+    F.DirichletBC(surfaces=[1], value=0, field="solute")
+]
+
+initial_concentration = sp.Piecewise(
+    (C_0, F.x <= preloaded_length),
+    (0, True)
+)
+model.initial_conditions = [
+    F.InitialCondition(field="solute", value=initial_concentration)
+]
+
+model.materials = [F.Material(id=1, D_0=D, E_D=0)]
+
+model.T = F.Temperature(500)  # ignored in this problem
+
+model.dt = F.Stepsize(
+    initial_value=0.01,
+    stepsize_change_ratio=1.1,
```

Comment:
Maybe we should avoid changing the stepsize. Does that improve the agreement for x=12m?

Reply:
I tested this locally. Changing the stepsize doesn't do much but increase the mesh refinement helps with agreement:

![image](https://github.com/user-attachments/assets/16ce97f4-d4ea-474b-91ec-11b507943052)

This is with 
```python
vertices = np.concatenate([
    np.linspace(0, 10, 400),
    np.linspace(10, 100, 1000),
])
```